internal/subscription/core.go

```diff
@@ -344,6 +345,70 @@ func createDelayTopics(ctx context.Context, m *Model, topicCore topic.ICore, top
 	return nil
 }
 
+// RescaleSubTopics - Get all the subs and rescale all the Retry/Delay/DLQ topics
+func (c *Core) RescaleSubTopics(ctx context.Context, topicModel *topic.Model) error {
+	subList, err := c.List(ctx, Prefix+topicModel.ExtractedProjectID)
+	if err != nil {
+		return err
+	}
+
+	for _, m := range subList {
```

Comment:
Subscriptions for topics can belong to any project. WHy are we restricting to only the current project?